core/frontend/services/redirects/settings.js

```diff
@@ -142,8 +142,14 @@ const setFromFilePath = (filePath, ext) => {
         });
 };
 
+const defaultJsonFileContent = '[]';
```

Comment:
Would rather name `defaultJsonFileContent` as `defaultRedirectsContent`

Reply:
I think `defaultJsonFileContent` is the right name. Because json redirects file is the default until it is deprecated. 

The current default content for json file is the `[]`.

Reply:
I reviewed it today and your name sounds better. I'll change it that way.

Reply:
And the default file content for 'yaml' file is the empty string, not `[]`.